StPicoEvent/StPicoEvent.h

```diff
@@ -110,6 +111,8 @@ class StPicoEvent : public TObject
   UShort_t mFillId;
   Float_t  mBField;
 
+  Int_t    mTime;//To set timestamp for St_db_Maker. This is what StMuDstMaker used to GetEvtHddr()->SetGMTime(cast into unsigned int). Peifeng Liu
```

Comment:
I found that if I don't set a timestamp in code reading picodst and then use the St_db_Maker, the code would crash with St_db_Maker complaining no timestamp. So should I just set a somehow arbitrary timestamp (like 20160101,0) in the macro?

Also, I don't see any mechanism for the current PicoDst to send runid information to other makers. Should that be implemented?